query/src/main/scala/filodb/query/exec/PeriodicSamplesMapper.scala

```diff
@@ -33,7 +33,9 @@ final case class PeriodicSamplesMapper(startMs: Long,
                                        stepMultipleNotationUsed: Boolean = false,
                                        funcParams: Seq[FuncArgs] = Nil,
                                        offsetMs: Option[Long] = None,
-                                       rawSource: Boolean = true) extends RangeVectorTransformer {
+                                       rawSource: Boolean = true,
+                                       inclusive: Boolean = false
```

Comment:
Can we explore adding 1 to the window size during planning instead of this attribute ? Tp be done once it is confirmed that it is truly needed.

Reply:
is it for performance reasons? adding 1 seems to be hacky, maybe we can subclass PeriodicSamplesMapper (though not sure if it will speed it up).

Reply:
For one, it reduces complexity of PeriodicSamplesMapper which is already complicated. 
Next, adding 1 to lookback is easier to read and understand.

Reply:
The other way of doing it is to add 1 locally in periodicSamplesMapper and retain old logic instead of changing the algorithms

Reply:
Anyway, please see other thread and re-evaluate if this change is absolutely needed.

Reply:
added 1 to the window